zork++/src/lib/compiler/mod.rs

```diff
@@ -474,7 +471,7 @@ mod sources {
 /// kind of workflow that should be done with this parse, format and
 /// generate
 mod helpers {
-    use crate::bounds::{ExtraArgs, TranslationUnit};
+    use crate::project_model::{ExtraArgs, TranslationUnit};
```

Comment:
This is the unique thing that I don't like from all the changes in the overall. Traits should be higher abstractions, not being coupled with domain models